app/controllers/questions_controller.rb

```diff
@@ -70,6 +70,6 @@ def set_question
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def question_params
-      params.require(:question).permit(:text, :x_weight, :y_weight)
+      params.require(:question).permit(:x_weight, :y_weight, :text)
```

Comment:
Why

Reply:
Uhm @spencerldixon I didn't intentionally change this file, I thought I just merged with master but maybe there was a conflict and I merged it wrongly, I will restore it back.